### DISCUSSION TITLE
Is Storybook (Vite Builder) possible without using babel ?

### DISCUSSION THREAD

**Discussion by melon2504** [storybookjs/storybook#26542] [Help]
### Summary

Currently only Storybook + Webpack uses SWC.

Is it possible for having **Storybook + Vite builder** use SWC and not babel.

E.g. vite provides @vitejs/plugin-react-swc which replaces babel, so any similar package/add-on available that can do this ?

**Comment by valentinpalkovic** [comment]
Hi @melon2504 

Since Storybook 8, we are not initializing any kind of default Vite plugins. Storybook picks up the plugins which are defined in your Vite configuration. If you are using `@vitejs/plugin-react-swc` in your Vite configuration, Storybook will pick it up and uses it as well!
